servers/services-bench/README.md

```diff
@@ -0,0 +1,28 @@
+# Nessie VersionStore micro benchmarks
+
+Building:
+```bash
+./gradlew :nessie-services-bench:jmhJar
+```
+
+Running:
+```bash
+java -jar servers/services-bench/build/libs/nessie-services-bench-*-jmh.jar
+```
+
+## Async-profiler
+
+See the [Async Profiler repo](https://github.com/async-profiler/async-profiler) for a pre-built library or how to build
+it from source.
+
+Running (Linux):
+```bash
+LD_LIBRARY_PATH=(PATH-TO-LIBRARY)/ java \
```

Comment:
FYI, the only command that worked for me on macOS was:

```shell
java -Djava.library.path=/path/to/async-profiler-2.9-macos/build/ -jar servers/services-bench/build/libs/nessie-services-bench-*-jmh.jar -prof async
```

Neither `LD_LIBRARY_PATH` nor `DYLD_LIBRARY_PATH` worked.

Reply:
Updated README